server/enterprise/message_export/actiance_export/actiance_export.go

```diff
@@ -119,6 +121,11 @@ type FileUploadStartExport struct {
 	FilePath        string   `xml:"FileName"`     // the path to the file, as stored on the server
 }
 
+func (f FileUploadStartExport) SortVal() (int64, string) {
+	// file messages should come after the message they were with
+	return f.UploadStartTime, strings.Repeat("z", 26)
```

Comment:
personally, I thought specifying the length of the string made it more explicit that we're making a 26-char id of all z's (saving you the trouble of counting it in case you wanted to double check)

Reply:
Not a problem, but since we are reusing it in a couple of places it could be a package-level variable. It has the bonus of having a descriptive name and not needing to recalculate at runtime.

Reply:
Ahh, understood. Good idea, done!